README.md

```diff
@@ -11,7 +11,9 @@ In `mix.exs`, add the ExMachina dependency:
 
 ```elixir
 def deps do
-  [{:ex_machina, "~> 0.4"}]
+  [
+    {:ex_machina, "~> 0.4", only: "test"}
```

Comment:
@paulcsmith I really wanted to add a trailing comma here and elsewhere – it's a strongly preferred style for me for many reasons – but I suspect you don't want that here? I don't think it's idiomatic in Elixir, sadly.

I checked thoughtbot's "guides" repo, and it *is* in your Ruby guide, but you don't have an Elixir one…